scripts/Source/sslThreadLibrary.psc

```diff
@@ -223,7 +225,7 @@ Actor[] function SortActorsByAnimation(actor[] Positions, sslBaseAnimation Anima
 		if (Futas[0] + Futas[1]) < 1 && (Genders[0] == ActorCount || Genders[1] == ActorCount)
 			return Positions ; Nothing to sort
 		elseIf Animation && Animation != none && !Animation.HasTag("Futa")
-			if Animation.Males == Animation.PositionCount || Animation.Females == Animation.PositionCount || Genders == Animation.Genders
+			if Animation.Males == Animation.PositionCount || Animation.Females == Animation.PositionCount ;|| Genders == Animation.Genders
```

Comment:
Fix issue with the SortActorsByAnimation function caused by try to reduce the time

the original idea was escape the function if the positions genders are the same as in the animation but need to check position by position instead of the Genders == Animation.Genders so for now is on my TODO list.